RedHat-UX/red-hat-design-system
# docs(menu): hide `<rh-menu>` from sidenav and all elements

## What I did

1. Hid `<rh-menu>` link from "Elements" side navigation on ux.redhat.com.
1. Hid `<rh-menu>` from Elements > All Elements page.
~~1. Added a new file, `hiddenItems.yaml`, for elements that we don't want to have shown in these nav menus.~~
1. Added ~~two~~ one new `type` value (`hidden` ~~and `coming-soon`~~) to `tagName` in `repoStatus.yaml`.
~~1. Moved `comingSoonItems.yaml` into `repoStatus.yaml` and removed `comingSoonItems.yaml`.~~

## Testing Instructions

1. Load [DP](https://deploy-preview-1988--red-hat-design-system.netlify.app/elements/).
3. Marvel in the glory about what a successful build can accomplish.
4. Expand the Elements menu in the side navigation.
5. Do you see the word "Menu" between "Icon" and "Navigation (primary)"?
6. If no, proceed to the next step.
7. Click [All Elements](https://deploy-preview-1988--red-hat-design-system.netlify.app/elements/).
8. In the list of Elements (made with `<rh-tile>`'s), look for Menu.
9. Don't see menu? Then we're good there.
10. Ensure `<rh-footnote>` and `<rh-jump-links>` appear in the All Elements list of tiles, but are disabled tiles.
11. Ensure footnote and jump links do _not_ show up as links in the side navigation.
12. All Yes? Then ✅ 

## Notes to Reviewers

Closes #1980.
